cmd/var.go

```diff
@@ -20,28 +20,33 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
-// Client is Clientset to use for specified cluster
-var Client *kubernetes.Clientset
-var err error
-var kubeconfig *string
-
-// KUBECONFIG contains path to file
-var KUBECONFIG string
-var masterURL *string
-var garden bool
-var seed bool
-var project bool
-
-// configuration for gardenctl
-const suggestionsMinimumDistance int = 2
-const prefixMatching bool = true
-
-// credentials
-var username string
-var password string
-
-// file pathes
+const (
+	// configuration for gardenctl
+	suggestionsMinimumDistance int  = 2
+	prefixMatching             bool = true
+
+	// NamespaceForProject contains the namespace of a project
+	namespaceForProject string = "project.garden.sapcloud.io/namespace"
```

Comment:
This label/annotation is removed with gardener/gardener#1035. Can't we use `.spec.namespace`?